ui/perfherder/compare/CompareTable.jsx

```diff
@@ -50,22 +51,24 @@ export default class CompareTable extends React.PureComponent {
 
   retriggerJobs = async (results, times) => {
     // retrigger base revision jobs
+    const { projects } = this.props;
+
     this.retriggerByRevision(
       results.originalRetriggerableJobId,
-      results.originalRepoName,
+      RepositoryModel.getRepo(results.originalRepoName, projects),
       true,
       times,
     );
     // retrigger new revision jobs
     this.retriggerByRevision(
       results.newRetriggerableJobId,
-      results.newRepoName,
+      RepositoryModel.getRepo(results.newRepoName, projects),
```

Comment:
I remember there was a reason that I didn't think I could call this everywhere I needed a Repository object.  I don't remember what the reason was.  Is that not an issue here?

Reply:
Well, you have to pass in the list of ALL repositories.  So it's a bit cumbersome compared to just using the ``currentRepo``.  In Perfherder, you may want to call against different repos.  But you're basically always on the same repo in Treeherder.  So just passing ``currentRepo`` to where you need it makes sense.